src/family_name_classifier.py

```diff
@@ -0,0 +1,315 @@
+import builtins
```

Comment:
Disclaimer: 💅

Python style guide propose [a convention to organize the `import` section](https://www.python.org/dev/peps/pep-0008/#imports). In a nutshell first we have native modules, then modules installed via `pip`, and then your application's modules.

That said, I would have written:

```python
import builtins
import collections
import itertools
import random
import re
import unicodedata

import numpy as np
import pandas as pd
import sklearn.cross_validation
import sklearn.externals.joblib
import sklearn.feature_extraction
import sklearn.linear_model
```

But… well… again: 💅